connector/src/main/java/org/geysermc/connector/entity/ItemEntity.java

```diff
@@ -36,15 +36,29 @@
 public class ItemEntity extends Entity {
 
     public ItemEntity(long entityId, long geyserId, EntityType entityType, Vector3f position, Vector3f motion, Vector3f rotation) {
-        super(entityId, geyserId, entityType, position, motion, rotation);
+        super(entityId, geyserId, entityType, position.add(0d, entityType.getOffset(), 0d), motion, rotation);
+    }
+
+    @Override
+    public void setMotion(Vector3f motion)
+    {
```

Comment:
Can you move this bracket up to the same line as the function declaration?

Reply:
Sorry for not following the coding standards. habit..